client/me/account/email-not-verified-notice.tsx

```diff
@@ -1,51 +1,30 @@
-import { __ } from '@wordpress/i18n';
-import { useDispatch, useSelector } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
 import Notice from 'calypso/components/notice';
 import NoticeAction from 'calypso/components/notice/notice-action';
-import { successNotice, errorNotice } from 'calypso/state/notices/actions';
-import { useSendEmailVerification } from '../../landing/stepper/hooks/use-send-email-verification';
+import { isCurrentUserEmailVerified } from 'calypso/state/current-user/selectors';
+import { useResendEmailVerification } from '../../landing/stepper/hooks/use-resend-email-verification';
 import './email-not-verified-notice.scss';
-import { isCurrentUserEmailVerified } from '../../state/current-user/selectors';
-
-const resendEmailNotice = 'resend-verification-email';
 
 const EmailNotVerifiedNotice = () => {
```

Comment:
I was surprised to see changes to this file. To confirm, I think what we did here is just move the logic and imports for handleResend() in this file to a new custom hook, use-resend-email-verification? 

Assuming so, one small question is where that hook should live. Right now, the new hook is in client/landing/stepper/hooks. Theoretically, I'd think hooks defined in that directory are stepper-specific, though I'm not sure if that's the case. It seems like it would make sense for hooks used in different folders of the client directory to live in some more generic location.